content/en/docs/gsoc/_index.md

```diff
@@ -10,7 +10,7 @@ Introduction
 ------------
 
 Navidrome started in February 2016 as a modern and lightweight alternative to [Subsonic](http://subsonic.org):
-written in [Go]( https://golang.org/ )/[React](https://reactjs.org/), implementing the [subsonic API]( http://www.subsonic.org/pages/api.jsp ) and thus compatible with all the [subsonic clients]( http://www.subsonic.org/pages/apps.jsp ) in the world, licensed under GPL3, … Being relatively young, it does come/use modern development practices like continuous integration, a comprehensive testsuite, a relatively clean codebase, automatic dependency upgrades, automatic linting/CI/static analysis/… on each pull-request, [comprehensive documentation]( https://www.navidrome.org/docs/ ) … It recently gained popularity due to the decay of [Subsonic](http://subsonic.org)/[Airsonic](http://airsonic.org), and currently has more than 4M downloads of its [docker image]( https://hub.docker.com/r/deluan/navidrome ), and had its binaries downloaded more than 12k times.
+written in [Go]( https://golang.org/ )/[React](https://reactjs.org/), implementing the [subsonic API]( http://www.subsonic.org/pages/api.jsp ) and thus compatible with all the [subsonic clients]( http://www.subsonic.org/pages/apps.jsp ) in the world, licensed under GPL3, … Being relatively young, Navidrome comes with the usage of modern development practices like continuous integration, a comprehensive testsuite, a relatively clean codebase, automatic dependency upgrades, automatic linting/CI/static analysis/ for each pull-request, [comprehensive documentation]( https://www.navidrome.org/docs/ )etc. Navidrome recently gained popularity due to the decay of [Subsonic](http://subsonic.org)/[Airsonic](http://airsonic.org), and currently has more than 5M downloads of its [docker image]( https://hub.docker.com/r/deluan/navidrome ), and has its binaries downloaded more than 12k times.
```

Comment:
"automatic linting/CI/static analysis/… on each pull-request," is correct.
And so is "and had its binaries downloaded more than 12k times."

Also, having "Navidrome" 3 times in a paragraph isn't super-duper nice in my opinion.

Reply:
Ohh, Okay. It's just that the first time I read through the page, I felt like something seemed to be off. 
Anyways, I guess I will just close this pr now.